src/hb-ot-glyf-table.hh

```diff
@@ -1039,6 +1042,62 @@ struct glyf
         add_gid_and_children (item.glyphIndex, gids_to_retain, depth);
     }
 
+    bool
+    get_path (hb_font_t *font, hb_codepoint_t gid,
+	      hb_ot_glyph_decompose_funcs_t *funcs, void *user_data) const
```

Comment:
Should mention learned from https://github.com/opentypejs/opentype.js/blob/4e0bb99/src/tables/glyf.js#L222 for this